include/openmc/cell.h

```diff
@@ -23,17 +23,21 @@ namespace openmc {
 // Constants
 //==============================================================================
 
-// TODO: Convert to enum
-constexpr int FILL_MATERIAL {1};
-constexpr int FILL_UNIVERSE {2};
-constexpr int FILL_LATTICE {3};
-
-// TODO: Convert to enum
-constexpr int32_t OP_LEFT_PAREN   {std::numeric_limits<int32_t>::max()};
-constexpr int32_t OP_RIGHT_PAREN  {std::numeric_limits<int32_t>::max() - 1};
-constexpr int32_t OP_COMPLEMENT   {std::numeric_limits<int32_t>::max() - 2};
-constexpr int32_t OP_INTERSECTION {std::numeric_limits<int32_t>::max() - 3};
-constexpr int32_t OP_UNION        {std::numeric_limits<int32_t>::max() - 4};
+enum CellFillType
+{
+  FILL_MATERIAL = 1,
+  FILL_UNIVERSE = 2,
+  FILL_LATTICE = 3
```

Comment:
Would removing the integer values here and relying on the defaults do any harm?

Reply:
Probably not, but I didn't want to peruse the whole code base to make sure. I should look into this for the individual enums though. One example where the numeric values of the enum are actually used is in the second commit of this PR, where an explicit cast to int is required in MPI mode. So... I'll be double-checking each enum to remove this.